keyboards/santoku/keymaps/shaymdev/keymap.c

```diff
@@ -0,0 +1,262 @@
+        case ALT_T(KC_O):
+        case GUI_T(KC_A):
+        case ALT_T(KC_SPC):
+            return TAPPING_TERM + 100;
+        default:
+            return TAPPING_TERM;
+    }
+}
+
+//#ifdef OLED_DRIVER_ENABLE
+#ifdef OLED_ENABLE
+void oled_task_user(void) {
+	// Vanity Text
+	static bool show_vanity_text = true;
+	if (show_vanity_text) {
+		oled_write_ln_P(PSTR("  Santoku Keyboard"), false);
+		oled_write_ln_P(PSTR("       by Tye"), false);
+		oled_write_ln_P(PSTR(""), false);
+		oled_write_ln_P(PSTR("    Hello, Shay."), false);
+		if (timer_read() > 7500) {
+			show_vanity_text = false;
+		}
+	}
+	else if (in_alttab ) {
+		oled_write_ln_P(PSTR("   ALT-TAB ACTIVE   "), true);
+
+	}
+	else {
+		uint8_t wpm = get_current_wpm();
+		if (wpm < 20) {
+			oled_write("      ", false);
+		}
+		else {
+			char wpm_display[9];
+			sprintf(wpm_display, "WPM:%d ", get_current_wpm());
+			oled_write(wpm_display, false);
+		}
+
+		// Host Keyboard Layer Status
+		switch (get_highest_layer(layer_state)) {
+			case _DVORAK:
+				oled_write_P(PSTR("  DVORAK\n"), false); //maybe just list the tap mods and combos
+				oled_write_ln_P(PSTR(""), false);
+				oled_write_ln_P(PSTR("ES  ',.py | fgcrl/"), false);
+				oled_write_ln_P(PSTR("TB  aoeui | dhtns-"), false);
+				oled_write_ln_P(PSTR("=   ;qjkx | bmwvz\\"), false);
+				break;
+			case _QWERTY:
+				oled_write_P(PSTR(" QWERTY\n"), false);
+				oled_write_ln_P(PSTR(""), false);
+				oled_write_ln_P(PSTR("TB  qwert | yuiop\\"), false);
+				oled_write_ln_P(PSTR("ES  asdfg | hjkl;'"), false);
+				oled_write_ln_P(PSTR("SH  zxcvb | nm,./"), false);
+				break;
+			case _LOWER:
+				oled_write_P(PSTR("   Lower   \n"), true);
+				oled_write_ln_P(PSTR(""), false);
+				oled_write_ln_P(PSTR(" `  !@{}& |  789-"), false);
+				oled_write_ln_P(PSTR("ES  #$()* |  456+"), false);
+				oled_write_ln_P(PSTR("SH  %^[]` |  123E"), false);
+				break;
+			case _RAISE:
+				oled_write_P(PSTR(" Raise \n"), true);
+				oled_write_ln_P(PSTR(""), false);
+				oled_write_ln_P(PSTR("Pr H ^ E PU|V+ F789"), false);
+				oled_write_ln_P(PSTR("In < V > PD|V- F456"), false);
+				oled_write_ln_P(PSTR("NL         |Mu F123"), false);
+				break;
+			case _ADJUST:
+				oled_write_P(PSTR("  Adjust  \n"), true);
+				oled_write_ln_P(PSTR(""), false);
+				oled_write_ln_P(PSTR("esc rst |"), false);
+				oled_write_ln_P(PSTR("        |  qw dv"), false);
+				oled_write_ln_P(PSTR("        |"), false);
+				break;
+			default:
+				oled_write_ln_P(PSTR("Undefined"), false);
+		}
+	}
+
+	/*
+	// Host Keyboard LED Status
+	led_t led_state = host_keyboard_led_state();
+	oled_write_P(led_state.num_lock ? PSTR("NUM ") : PSTR("    "), false);
+	oled_write_P(led_state.caps_lock ? PSTR("CAP ") : PSTR("    "), false);
+	oled_write_P(led_state.scroll_lock ? PSTR("SCR ") : PSTR("    "), false);
+	*/
+}
+#endif
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+	state = update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+	return state;
```

Comment:
This is a bit simpler: 
```suggestion
	return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
```

Reply:
Ah, yeah I had that separated because I was doing some stuff in between those two lines at one point.

Reply:
Figured as much, and no worries.